File: Modul 2/Soal 4/PRAK204-2310817210005-Galih_Aji_Sabdaraya.c
#include <stdio.h>

int main(){
    double r, t;
    scanf("%lf %lf", &r, &t);
    double pi = (double) 22/7;
    double Volume = pi * r * r * t;
    double Luas = 2 * pi * r * (r + t);
    double Keliling = 2 * pi * r;
    printf("\nVolume = %.2lf\n", Volume);
    printf("Luas = %.2lf\n", Luas);
    printf("Keliling = %.2lf", Keliling);
    
    return 0;
}

File: Modul 2/Soal 4/PRAK204-2310817210005-Galih_Aji_Sabdaraya.py
r =input().split()
pi = float(22/7)
if len(r) == 2:
    r, t = map(float, r)
elif len(r) == 1:
    r = float(r[0])
    t = float(input())
Volume = pi * r * r * t
Luas = 2 * pi * r * (r + t)
Keliling = 2 * pi * r
print(f"\nVolume = {'%.2f'}" %Volume)
print(f"Luas = {'%.2f'}" %Luas)
print(f"Keliling = {'%.2f'}" %Keliling)